kihuha/iReporter-db
# #162661711 - Validate endpoint URLs

#### What does this PR do?
Validates URLs for all endpoints

#### Description of the task to be completed
Show an error message when an invalid URL is entered. 

#### How should this be tested
After cloning the repo, navigate to the root directory on your terminal and start the server by typing python run.py. Using postman, send either a post or a get request to any endpoint with invalid data and you should receive a customized error message

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/162661711
